### DISCUSSION TITLE
How to display optimized and responsive images from Strapi as a provider ?

### DISCUSSION THREAD

**Discussion by ghost** [nuxt/image#720] [Q&A]
I can see many issues on my side concerning the integration [strapi as an image provider in my nuxt 3 project](https://v1.image.nuxtjs.org/providers/strapi).
First, the only attribute working with this provider is the “modifier”. We can set a unique breakpoint, something like this:

`<nuxt-img provider="strapi" src="..." :modifiers="{ breakpoint: 'small' }" />`

Secondly: The url generated is wrong, it’s something like **/uploads/large/my_image.png** instead of
**/uploads/large_my_image.png**

I can’t figure out how to make the image responsive depending on the screen size. Has anyone already used it before ?

Thank you!

**Comment by Baroshem** [comment]
Hey,

I believe this question should be asked on the https://github.com/nuxt/image as it is related to the image module :)
